awt/graphics.cs

```diff
@@ -2033,18 +2033,14 @@ internal NetGraphicsDevice(Screen screen)
 
         public override java.awt.GraphicsConfiguration[] getConfigurations()
```

Comment:
It looks like this methods is similar to getDefaultConfiguration (without S). Technically, we could replace calls ofgetConfigurations by getDefaultConfiguration, right ?

Reply:
getConfigurations() returns array of GraphicsConfiguration. In JDK it can return multiple configurations if a device support multiple Pixel Formats. However support of different Pixel Formats is not implemented in IKVM and is not required.
Therefore this array always has 1 element in IKVM.

getDefaultConfiguration() returns single instance of GraphicsConfiguration.